crates/settings/src/lib.rs

```diff
@@ -54,13 +54,16 @@ fn create_config_folder(path: &PathBuf) -> Result<PathBuf, std::io::Error> {
         std::fs::create_dir_all(path)?;
     }
     let filename = path.join("settings.toml");
-
+    let default_config = include_str!("../default/settings.toml");
     if filename.exists() {
         log::warn!("File exists");
         log::warn!("{:?}", filename);
     } else {
         log::warn!("File does not exist");
-        std::fs::copy("default/settings.toml", &filename)?;
+        // FIXME: the logic of settings that each app - desktop or server have their own default settings.toml
```

Comment:
@mre I want your opinion on how to design it better - see FIXME: comment.

Reply:
I think we could have an environment variable for the config path that could be overwritten.
Then we could use it here.

```sh
TERRAPHIM_CONFIG_PATH=...
``` 

It could be set to a sane default. 
Just an idea.

Reply:
What is the path variable for default cargo run and cargo test? We can put the default config into that path, and it will pick it up automatically for run and test, but then subfolders in the Axum server and desktop can have their own when run.